parse/src/main/java/org/dst/parse/util/CodeUtils.java

```diff
@@ -0,0 +1,23 @@
+package org.dst.parse.util;
+
+import org.apache.commons.jexl3.JexlContext;
+import org.apache.commons.jexl3.JexlEngine;
+import org.apache.commons.jexl3.JexlExpression;
+import org.apache.commons.jexl3.MapContext;
+import org.apache.commons.jexl3.internal.Engine;
+
+import java.util.Map;
+
+public class CodeUtils {
```

Comment:
What's the purpose of this class? Could you plz add a comment to explain about it?

Reply:
it's similar with reflectUtils